templates/environment.template.json

```diff
@@ -109,6 +109,14 @@
         "description": "The number of shared worker app subnets to provision."
       }
     },
+    "sharedSqlServerSubnetCount": {
```

Comment:
Could this be a more generic private endpoint subnet as we could have multiple private endpoints deployed to the same subnet? https://docs.microsoft.com/en-us/azure/private-link/private-link-faq#can-i-create-multiple-private-endpoints-in-same-vnet--can-they-connect-to-different-services--

Reply:
Good idea although the idea is in a future story we'll associate an NSG with this subnet and have ASG rules specific to SQL access

Reply:
Ah good point, as discussed I think maxValue for the subnetCount of 1 is likely suitable.